CHANGELOG.md

```diff
@@ -7,6 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Introduce `LakeConfigBuilder` for creating configs
+- Now you can provide AWS credentials through the `LakeConfig`
+  ```rust
+  let config = LakeConfigBuilder::default()
+    .s3_bucket_name("near-lake-data-testnet")
+    .start_block_height(88220926)
+    .with_custom_credentials(
```

Comment:
@khorolets There are still a few places in the comments and markdown files that mention `with_` prefix, please fix those